packages/grz-pydantic-models/src/grz_pydantic_models/submission/metadata/v1.py

```diff
@@ -521,6 +521,17 @@ def ensure_extension_matches_type(self):
                     )
         return self
 
+    @model_validator(mode="after")
+    def ensure_file_paths_are_relative(self):
+        file_path = Path(self.file_path)
+        if file_path.is_absolute():
+            raise ValueError(
+                "File paths must be relative to the submission root, "
+                "e.g.: sequencing_data/patient_001/patient_001_dna.fastq.gz; "
+                "symlinks are allowed."
```

Comment:
File paths must be relative to `files/` under the root, not root: https://github.com/BfArM-MVH/grz-tools/pull/171

Reply:
Good catch, copied that from the docstring in the model

Reply:
Fixed the wording where applicable, please re-check